common/src/serializer.rs

```diff
@@ -341,16 +341,16 @@ pub fn serialize_circuit(circuit: &Circuit) -> ConstraintSystem {
             Opcode::Directive(_) => {
                 // Directives are only needed by the pwg
             }
-            Opcode::Block(_, _) => {
-                // TODO: implement serialization of blocks to match BB's interface
```

Comment:
Barretenberg doesn't have these opcodes, so I think just saying they are unsupported is enough to note serialize it for now. Thry should get changed in the VM to opcodes that are supported by BB.

Reply:
I'd move `Opcode::Oracle` to the same case as `Opcode::Directive`. It's not a case of being unsupported - it simply doesn't lay constraints.